packages/web/src/pages/search-page-v2/SearchCatalogTile.tsx

```diff
@@ -0,0 +1,35 @@
+import { IconSearch, Paper, Text, useTheme } from '@audius/harmony'
+
+import { useMedia } from 'hooks/useMedia'
+const messages = {
+  cta: 'Search the Catalog',
+  description:
+    'Apply filters or search to discover tracks, profile, playlists, and albums.'
+}
+
+export const SearchCatalogTile = () => {
+  const { color } = useTheme()
+  const { isMobile } = useMedia()
+  return (
+    <Paper
+      pv={isMobile ? '2xl' : '3xl'}
```

Comment:
Is this the best way to do this? Lots of branching on `isMobile`, maybe I should just have a single branch

Reply:
yeah this is what we do in signup, what are your thoughts on the best way? one way i could see is to build a breakpoint option to harmony so you can pass object of styles to each prop?

Reply:
Yeah that's interesting although I don't think much more terse than what we have here. I wonder if this is where we could utilize `.styled`? Create styled components for mobile and then extend/overwrite those styles for desktop